L1Trigger/L1THGCal/python/customHistoSeeding.py

```diff
@@ -94,11 +89,18 @@ def custom_3dclustering_XYHistoMax(process,
                                    nBins_X1=histoMaxXYVariableDR_C3d_params.nBins_X1_histo_multicluster,
                                    nBins_X2=histoMaxXYVariableDR_C3d_params.nBins_X2_histo_multicluster,
                                    seed_threshold=histoMaxXYVariableDR_C3d_params.threshold_histo_multicluster,
-                                   seed_position=histoMaxXYVariableDR_C3d_params.seed_position,
+                                   seed_position=histoMaxXYVariableDR_C3d_params.seed_position
                                    ):
     parameters_c3d = histoMaxXYVariableDR_C3d_params.clone()
     set_histomax_seeding_params(parameters_c3d, nBins_X1, nBins_X2,
             histoMaxXYVariableDR_C3d_params.binSumsHisto,seed_threshold)
     process.hgcalBackEndLayer2Producer.ProcessorParameters.C3d_parameters.histoMax_C3d_seeding_parameters = parameters_c3d
     return process
 
+def custom_3dclustering_seedNoArea(process,
+                                   seeds_norm_by_area=cms.bool(False),
```

Comment:
Since the function is meant to remove the norm by area, this shouldn't be a parameter of the function. Otherwise we could call `custom_3dclustering_seedNoArea(process, seeds_norm_by_area=True)` which would be a bit inconsistent.